packages/dds/tree/src/test/domains/json/twitter.ts

```diff
@@ -0,0 +1,631 @@
+import { IRandom, makeRandom, SpaceEfficientMarkovChain } from "@fluid-internal/stochastic-test-utils";
+import {
+    createAlphabetFromUnicodeRange,
+    getRandomEnglishString,
+    getRandomNumberString,
+    getSizeInBytes,
+} from "./jsonGeneratorUtils";
+
+/**
+* This file contains logic to generate a JSON file that is statistically similar to the well-known
+* json benchmarks twitter.json - https://github.com/serde-rs/json-benchmark/blob/master/data/twitter.json
```

Comment:
As a small favor, would you mind changing the URL to a "raw" link?
```
https://raw.githubusercontent.com/serde-rs/json-benchmark/master/data/twitter.json
```
(The "raw" link is easy to use with wget or curl to download the original for comparison.)

Reply:
Done.